src/fmu/dataio/export/rms/volumetrics.py

```diff
@@ -0,0 +1,283 @@
+                "Including zone maps is currently not implemented"
+            )
+        if self.include_total_maps:
+            raise NotImplementedError(
+                "Including total maps is currently not implemented"
+            )
+        if self.include_3dgrid_properties:
+            raise NotImplementedError(
+                "Including 3D parameters is currently not implemented"
+            )
+
+    def _warn_if_forcefolder(self) -> None:
+        if self.forcefolder:
+            warn(
+                "A 'forcefolder' is set. This is strongly discouraged and will be "
+                "removed in coming versions",
+                FutureWarning,
+            )
+
+    def _export_volume_table(self) -> dict[str, str]:
+        """Do the actual volume table export using dataio setup."""
+        # if self._dataframe.is_empty():
+        #     raise ValueError("No data in volume table.")
+
+        edata = dio.ExportData(
+            config=self._global_config,
+            content="volumes",
+            unit="m3" if self._units == "metric" else "ft3",
+            vertical_domain={"depth": "msl"},
+            workflow=self.workflow,
+            forcefolder=self.forcefolder,
+            classification=self.classification,
+            tagname=self.tagname,
+            name=self.name if self.name else f"{self.grid_name}_volumes",
+            rep_include=False,
+        )
+
+        out = edata.export(self._dataframe)
+        _logger.debug("Volume result to: %s", out)
+        return {"volume_table": out}
+
+    def export(self) -> dict[str, str]:
+        """Export the volume table."""
+        return self._export_volume_table()
+
+
+def export_rms_volumetrics(
+    project: Any,
+    grid_name: str,
+    volume_job_name: str,
+    global_config: str | Path | dict = "../../fmuconfig/output/global_variables.yml",
+    forcefolder: str = "",  # unsure if we shall allow this?
+    subfolder: str = "",
+    name: str = "",
+    tagname: str = "",
+    classification: str = "restricted",
+    workflow: str = "rms volumetric run",
+    include_zone_maps: bool = False,
+    include_total_maps: bool = False,
+    include_3dgrid_properties: bool = False,
+) -> dict[str, str]:
+    """Simplified interface when exporting volume tables (and assosiated data) from RMS.
+
+    As the export_volumetrics may have multiple output (storing both tables, maps and
+    3D grids), the output from this function is always a dictionary. The table is
+    mandatory output, while maps and 3D grid data are optional (not yet implemented).
+
+    Args:
+        project: The 'magic' project variable in RMS.
+        grid_name: Name of 3D grid model in RMS.
+        volume_job_name: Name of the volume job.
+        global_config: Optional. The global config can either point to the
+            global_variables file, or it can be a dictionary. As default, it assumes
+            a path which is the current standard in FMU.
```

Comment:
Could be good to display this default path 🙂 

```suggestion
           '../../fmuconfig/output/global_variables.yml' which is the current standard in FMU.
```